Automattic/wp-calypso
# Assembler: Fix the page patterns are not rendered

Related to https://github.com/Automattic/wp-calypso/pull/83736#issuecomment-1790065399

## Proposed Changes

* Resolve the page patterns are not rendered as those patterns are excluded from the `patternsMapByCategory`

## Testing Instructions



* Go to the Assembler
* Continue to the page screen
* Select any pages
* Ensure the preview of pages is rendered correctly

## Pre-merge Checklist
